### ISSUE TITLE
Support sdk v4

### ISSUE THREAD

**Issue by lekoala** [phptek/silverstripe-sentry#94]
The v4 of the sdk has been published end of last year. I was wondering if you were planning to support this?

https://github.com/getsentry/sentry-php/blob/4.0.0/UPGRADE-4.0.md

it seems at first glance that it should even be possible to support both v3/v4 in the same package

one thing i liked in the v4 is the ability to configure better the http client (for example, when testing locally, i realized the calls were not made because of the missing ca.info in my php config)

**Comment by phptek** [comment]
Thanks for raising this @lekoala. If I have time, I'll look into it. So this is just a courtesy message to acknowledge the ticket at this point.

Thanks a lot :-)

**Comment by phptek** [comment]
@lekoala From what you can see, does this PR appear to support the version you're thinking about? https://github.com/phptek/silverstripe-sentry/pull/90

**Comment by lekoala** [comment]
yes indeed. i don't know if it's better to upgrade only to v4 or to do ^3 |^4 . From what i can see there is no downsides in using the updated sdk. I'll make a comment with some suggestions on the pr from some things i've realized when upgrading to the v4 in another project not using your module
